docs/team-process.md

```diff
@@ -2,16 +2,63 @@
 
 Now that you've logged in and played around with our Dawson system a bit on a deployed environment, let's talk about our team process and what is expected from our fellow teammates.  This part of the documentation should help members get on the same page when it comes to the soft-skills required to work as a well-oiled agile machine.
 
-## 🤝🏿 Working Agreement
+## Onboarding
+
+Here are some few resources for documentation associated with onboarding:
+
+1. [Onboarding checklist](https://docs.google.com/document/d/1zhvp8vcWnVSvUbTKZ-0sHrf5lA6OTlpXqvjeLgKJtVk/edit#)
+2. [Onboarding Reference Guide](https://docs.google.com/document/d/1jOd0wAsqTDBKsFeEvUj9ezvdTimTsqTwkYd6Imibl3c/edit#)
```

Comment:
:smiley: Thanks for adding these! I might remove the `/edit#`, but I'm not sure that actually changes anything.